lib/commands/context.js

```diff
@@ -370,12 +370,16 @@ extensions.onPageChange = async function onPageChange (pageChangeNotification) {
       .map((b) => b.trim())
       .filter((b) => !_.isEmpty(b));
     for (const blacklistedHost of blacklistedHosts) {
-      if (blacklistedHost === url && url === 'about:blank') {
+      if (blacklistedHost === 'about:blank' && url === 'about:blank') {
         return true;
       } else {
-        const hostname = new URL(url).hostname;
-        if (hostname === blacklistedHost) {
-          return true;
+        try {
+          const hostname = new URL(url).hostname;
+          if (hostname === blacklistedHost) {
+            return true;
+          }
+        } catch (ign) {
+          // do nothing if invalid URL
```

Comment:
I would rather print a warning here

Reply:
I don't know if a warning would be necessary. The URL comes from webkit and if its an invalid URL that webkit is giving us, then it's safe to just assume it doesn't match any of the blacklisted urls.